pkgs/applications/science/electronics/opensta/default.nix

```diff
@@ -0,0 +1,68 @@
+{ lib
+, stdenv
+, cmake
+, flex
+, bison
+, swig4
+, fetchFromGitHub
+, fetchurl
+, tcl
+, tclreadline
+, zlib
+}:
+
+let
+  cudd = stdenv.mkDerivation rec {
+    pname = "cudd";
+    version = "3.0.0";
+
+    src = fetchurl {
+      url = "https://github.com/davidkebo/cudd/raw/main/cudd_versions/cudd-${version}.tar.gz";
+      hash = "sha256-uOlmtFYslqA+f76iOXKVh9ezldU8rcw5pyA7Sc9+62k=";
+    };
+
+    meta = with lib; {
+      description = "Package for manipulation of binary and other kinds of decision diagrams";
+      homepage = "https://davidkebo.com/cudd/";
+      license = licenses.bsd3;
+    };
+  };
+in stdenv.mkDerivation rec {
+  pname = "opensta";
+  # project doesn't seem to make formal releases, this is the date of last commit
+  version = "20240321";
```

Comment:
```suggestion
  version = "0-unstable-2024-03-21";
```

Reply:
Is this the nixpkgs custom? I wasn't sure what to put in.

Reply:
Yes, see https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming

Reply:
Ah, thanks